runtime/queries/metricsview_timeseries.go

```diff
@@ -119,6 +119,9 @@ func (q *MetricsViewTimeSeries) buildMetricsTimeSeriesSQL(mv *runtimev1.MetricsV
 			whereClause += fmt.Sprintf(" AND %s < ?", timestampColumnName)
 			args = append(args, q.TimeEnd.AsTime())
 		}
+	} else {
+		return "", nil, fmt.Errorf("time dimension is missing")
+
```

Comment:
We can remove this. The group by `timeCol` will break anyway. This PR adds early failure for querying timeseries without time dim: https://github.com/rilldata/rill-developer/pull/1516